static/src/js/components/SecondaryToolbar/SecondaryToolbar.jsx

```diff
@@ -125,16 +128,31 @@ class SecondaryToolbar extends Component {
     } = this.props
 
     const { disableSiteTour } = getApplicationConfig()
-
     const { first_name: firstName = '' } = ursProfile
 
     const loggedIn = authToken !== ''
     const returnPath = window.location.href
+    const { pathname, search } = location
+    let isMapOverlay = false
+    let needsOverlayPaths = ['/search']
+
+    // Add map overlay for routes that utilze the map
```

Comment:
~utilze~ utilize

[Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) is an invaluable VSCode extension, if you dont have it installed